src/snyk/common/services/cliConfigService.ts

```diff
@@ -1,10 +1,18 @@
 import { ISnykApiClient } from '../api/apiСlient';
+import { Configuration } from '../configuration/configuration';
 
 export type SastSettings = {
   sastEnabled: boolean;
+  localCodeEngine: {
+    enabled: boolean;
+  };
 };
 
 export async function getSastSettings(api: ISnykApiClient): Promise<SastSettings> {
-  const { data } = await api.get<SastSettings>('cli-config/settings/sast');
+  const { data } = await api.get<SastSettings>('cli-config/settings/sast', {
+    headers: {
+      'x-snyk-ide': `vsc-${await Configuration.getVersion()}`,
```

Comment:
Didn't we say this should be the same as Amplitude?

Reply:
Update to be consistent with Snyk Code API, NPM test API and authentication UTM parameter ✅ Thanks for pointing it out!